src/CalculatorUITestFramework/StandardCalculatorPage.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Appium.Windows;
+using OpenQA.Selenium.Remote;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculatorUITestFramework
+{
+    /// <summary>
+    /// This class contains the UI automation objects and helper methods available when the Calculator is in Standard Mode.
+    /// </summary>
+    public class StandardCalculatorPage : ApplicationBase
+    {
+        public static StandardOperatorsPanel StandardOperators = new StandardOperatorsPanel();
+        public static MemoryPanel MemoryPanel = new MemoryPanel();
+        public static HistoryPanel HistoryPanel = new HistoryPanel();
+
+        public static void StandardCalculatorSetup(TestContext context)
```

Comment:
I don't think StandardCalculatorPage should inherit from ApplicationBase, and it follows that ApplicationSetup should get moved elsewhere (into the test code, probably). After that, this method should get renamed. Suggested name: `public static void NavigateToStandardCalculator()`